docs/docs-ref-autogen/office/office.document.yml

```
@@ -3129,9 +3129,9 @@ items:
       specified `data` is inserted at the insertion point</td><td>If `data` is a string, the specified text is
       inserted.</td></tr> <tr><td></td><td></td><td>If `data` is an array of arrays ("matrix") or a TableData object, a
       new Word table is inserted.</td></tr> <tr><td></td><td></td><td>If `data` is HTML, the specified HTML is inserted.
-      (Important: If any of the HTML you insert is invalid, Word won't raise an error. Word will insert as much of the
-      HTML as it can and omits any invalid data).</td></tr> <tr><td></td><td></td><td>If `data` is Office Open XML, the
-      specified XML is inserted.</td></tr> <tr><td></td><td></td><td>If `data` is a base64 encoded image stream, the
+      (**Important**: If any of the HTML you insert is invalid, Word won't raise an error. Word will insert as much of
+      the HTML as it can and omits any invalid data).</td></tr> <tr><td></td><td></td><td>If `data` is Office Open XML,

```

```suggestion
      the HTML as it can and will omit any invalid data).</td></tr> <tr><td></td><td></td><td>If `data` is Office Open XML,
```
Making verbs parallel

---

You could also go with "inserts...omits"